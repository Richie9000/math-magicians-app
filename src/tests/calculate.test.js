import operate from '../components/logic/operator';

 