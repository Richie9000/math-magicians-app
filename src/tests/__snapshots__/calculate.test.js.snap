// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`calculate snapshot test 1`] = `
Object {
  "next": null,
  "operation": null,
  "total": "12",
}
`;
